scripts/Engineering/EnginX.py

```diff
@@ -123,6 +123,7 @@ def run(ceria_run, do_cal, do_van, van_run, calibration_directory, calibration_g
     # if the calibration files that this run would create are not present, or the user has requested it, create the
     # calibration files
     if not all(expected_cals_present) or do_cal:
+        print("create calibration")
```

Comment:
I'm not sure how frequently you print out info in these scripts, but if there is a lot then this seems like it could be clearer. 1) Present tense over an imperative E.g. "creating calibration" 2) Why you're creating calibration E.g. "Calibration files are not present. Creating calibration..."
If this is the style throughout the EnginX scripts then ignore me

Reply:
honestly there aren't many prints in the script at all, these ones were mostly present so that whoever reviewed the pr could see the correct route was being followed, I could remove the prints now?

Reply:
If you don't think the prints are necessary for the users then I would remove